Images_And_Graphics_style.css_v1

```diff
@@ -387,12 +464,16 @@ ul li:hover ul {
   z-index: 99; /* Make sure it does not overlap */
   border: none; /* Remove borders */
   outline: none; /* Remove outline */
-  background-color: #657CCA; /* add a background color */
-  color: #000; /* color of text */
-  cursor: pointer; /* setting the icon to be used for my button on hover. https://www.w3schools.com/cssref/trvit.php?filename=trvcss_cursor. */
-  padding: 15px; /* add padding */
-  border-radius: 10px; /* Rounded corners */
-  font-size: 18px; /* font size */
+  background-color: #657CCA; /* Add a background color */
+  color: #000; /* Color of text */
+  cursor: pointer; /* Pointer cursor on hover */
+  width: 80px; /* Set button width to 80px */
+  height: 80px; /* Set button height to 80px */

```

Comment:
I added width and height and adjusted it to 80px so I could add a back to top emoji so students/caregiver will understand what the emoji means. so they can press the button and it takes the, to the top of the page.